pyresample/test/test_utils.py

```diff
@@ -198,6 +198,12 @@ def test_proj4_str_dict_conversion(self):
         self.assertIsInstance(proj_dict['lat_1'], float)
         self.assertIsInstance(proj_dict2['lat_1'], float)
 
+        # Value-less south parameter
+        proj_str = "+ellps=WGS84 +no_defs +proj=utm +south +type=crs +units=m +zone=54"
+        proj_dict = utils.proj4.proj4_str_to_dict(proj_str)
+        proj_str2 = utils.proj4.proj4_dict_to_str(proj_dict, sort=True)
+        self.assertEqual(proj_str, proj_str2)
```

Comment:
Could this be put in a separate test function/method?
Also, I don't remember the code that is exercised here, but could you summarize how this test makes sure None value is given to `south`?

Reply:
> Could this be put in a separate test function/method?

Done

> Also, I don't remember the code that is exercised here, but could you summarize how this test makes sure None value is given to `south`?

The test didn't check for the value associated with `south` in the dict (but was merely checking a `proj4_str_to_dict` -> `proj4_dict_to-str` round trip), but I added explicit assertions so now it does.